src/bikes/FoldingBike.java

```diff
@@ -0,0 +1,46 @@
+package bikes;
+
+import bikes.abstractbikes.Bike;
+
+import java.util.Objects;
+
+public class FoldingBike extends Bike {
+
+    public static final String BIKE_TYPE = "FOLDING BIKE";
+
+    private final String info;
+
+    private final String wheelsSize;
+
+    private final String numberOfGears;
+
+    public FoldingBike(String brand, String wheelsSize, String numberOfGears, String weight,
```

Comment:
Isn't it too many parameters in the constructor? Much more, they all are String objects. In which order I should add them? Can we refactor this?